spaces-config-service-impl/src/test/java/org/hypertrace/space/config/service/SpaceConfigRuleConverterImplTest.java

```diff
@@ -0,0 +1,75 @@
+package org.hypertrace.space.config.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import com.google.protobuf.ListValue;
+import com.google.protobuf.Struct;
+import com.google.protobuf.Value;
+import java.util.Map;
+import org.hypertrace.spaces.config.service.v1.AttributeValueData;
+import org.hypertrace.spaces.config.service.v1.SpaceConfigRule;
+import org.junit.jupiter.api.Test;
+
+class SpaceConfigRuleConverterImplTest {
+
+  private static final SpaceConfigRuleConverterImpl CONVERTER =
+      new SpaceConfigRuleConverterImpl();
+
+  private static final SpaceConfigRule RULE =
+      SpaceConfigRule.newBuilder()
+          .setId("test-id")
+          .setAttributeValueData(
+              AttributeValueData.newBuilder()
+                  .setAttributeKey("attrKey")
+                  .setAttributeScope("attrScope"))
+          .build();
+
+  private static final Value VALUE =
```

Comment:
This is defined in 3 places. Can we share this ?

Reply:
This is test data - I'd rather not share it between tests to keep things explicit and tests decoupled.